docs/content/v0.20/packaging.md

```diff
@@ -21,13 +62,15 @@ ends with `/github.com/kubestellar/kubestellar`.
 
 ### Derived files
 
-Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub. A contributor can use the individual `make` commands mentioned below to update subsets, or use the following command to update them all.
+Some files in the kubestellar repo are derived from other files there. These are all generated by contributors and kept in GitHub.
```

Comment:
"by contributors" is not clear to me . What does it mean ?  Everything in our repo is "generated by contributors" what is the difference ?

Reply:
Reworded.